src/policies/alpha_vector.jl

```diff
@@ -51,3 +51,43 @@ function Base.push!(p::AlphaVectorPolicy, alpha::Vector{Float64}, a)
     push!(p.alphas, alpha)
     push!(p.action_map, a)
 end
+
+function action(p::AlphaVectorPolicy, b)
+    return action(p, DiscreteBelief(p.pomdp, belief_vector(p, b)))
+end
+
+
+"""
+Given a belief, return a vector representation bv where bv[i] is the probability
+of being in the i-th state 
+    belief_vector(p::AlphaVectorPolicy, b)
+"""
+function belief_vector(p::AlphaVectorPolicy, b)
```

Comment:
I agree that having a `DiscreteBelief(m::POMDP, b)` is more elegant I will make the change. Then I am not sure if the specific methods for `AbstractParticleBelief` are needed. 
I suggest this 
```julia 
function DiscreteBelief(m::POMDP, b)
    bv = Vector{Float64}(n_states(m))
    for (i, s) in enumerate(ordered_states(m))
        bv[i] = pdf(b, s)
    end
    return DiscreteBelief(m, bv)
end
```

Then the following method should work fine with a particle belief: 
```julia
function action(p::AlphaVectorPolicy, b)
    return action(p, DiscreteBelief(p.pomdp, belief_vector(p, b)))
end
```
It would also remove the dependency with ParticleFilters.jl. 

Note: it implies getting rid of these functions: `value(p::AlphaVectorPolicy, b::AbstractParticleBelief)`, `action(p::AlphaVectorPolicy, b::AbstractParticleBelief)`, and `unnormalized_util(p::AlphaVectorPolicy, b::AbstractParticleBelief)`, Are they really useful? It seems to me that converting the particle belief to a discrete belief first is the right way to compute the value and the action at this belief point.